src/components/radio/radio.module.scss

```diff
@@ -44,24 +42,36 @@
   display: flex;
   width: 16px;
   height: 16px;
+  min-width: 16px;
   box-sizing: border-box;
   margin: auto 10px auto 0;
   border-radius: 100%;
   border-width: 1px;
   border-style: solid;
   line-height: 18;
+  border-color: var(--rp-ui-kit-color-field-border-2);
 
   &::after {
+    width: 8px;
+    height: 8px;
     content: '';
     display: block;
     border-radius: 100%;
     margin: auto;
-    background-color: var(--rp-ui-kit-topaz-pressed);
+    background-color: var(--rp-ui-kit-color-radio-cheked);
     opacity: 0;
     transform: scale(0.5);
     transition: transform 0.2s, opacity 0.2s;
   }
 
+  &:hover:not(.disabled) {
+    border-color: var(--rp-ui-kit-color-field-border-2-hover);
+
+    &::after {
+      background-color: var(--rp-ui-kit-color-radio-cheked-hover);
```

Comment:
Can we use `--rp-ui-kit-color-primary-hover` here to avoid creating the new var?